project-koku/koku
# [COST-1702] Traceback when using * with tag filtering on Openshift key values.

Fixes [COST-1702](https://issues.redhat.com/browse/COST-1702)

Changes proposed in this PR:
Fixing a 500 error when filtering ocp tags with value="*"



Testing Instructions:
1. Run `make create-test-customer` ...
2. Ingest this data `make load-test-customer-data` ...
3. Hit this endpoint [ GET http://localhost:8000/api/cost-management/v1/tags/openshift/version/?filter[value]=*]
4. Expect to see all data, see example below 
5. JSON 

```
{
    "meta": {
        "count": 16,
        "filter": {
            "value": [
                "*"
            ],
            "enabled": true,
            "time_scope_value": "-10",
            "time_scope_units": "day",
            "resolution": "daily"
        },
        "key_only": false,
        "group_by": {},
        "order_by": {},
        "access": {}
    },
    "links": {
        "first": "/api/cost-management/v1/tags/openshift/version/?filter%5Bvalue%5D=%2A&limit=100&offset=0",
        "next": null,
        "previous": null,
        "last": "/api/cost-management/v1/tags/openshift/version/?filter%5Bvalue%5D=%2A&limit=100&offset=0"
    },
    "data": [
        "Andromeda",
        "beta",
        "blue",
        "Development",
        "gamma",
        "green",
        "Mars",
        "master",
        "MilkyWay",
        "prod",
        "Production",
        "QA",
        "red",
        "Sombrero",
        "Test",
        "yellow"
    ]
}
```

---
